frontend/dashboard/components/FavoriteReposList/FavoriteReposList.tsx

```diff
@@ -3,17 +3,22 @@ import { RepoList } from '../RepoList';
 import { useTranslation } from 'react-i18next';
 import { useStarredReposQuery } from '../../hooks/queries';
 import { Heading } from '@digdir/designsystemet-react';
+import { TableSortStorageKey } from '@studio/components-legacy';
 
 export const FavoriteReposList = () => {
   const { t } = useTranslation();
-  const { data: userStarredRepos, isPending } = useStarredReposQuery();
+  const { data: userStarredRepos = [], isPending } = useStarredReposQuery();
```

Comment:
We should do "name-casting" to make a more readable variable. 😊 
```suggestion
  const { data: userStarredRepos = [], isPending: isPendingStarredRepos } = useStarredReposQuery();
```